docs/index.md

```diff
@@ -14,8 +14,8 @@ In here should go a nice description about CCR and what it's intended to do.
 ::: slot home-banner
 
 ::: danger HEADS UP
-  CCR is currently in the initial stages of development and isn't ready for deployment or testing just yet.  
-  Check out our  [contribution guidelines](https://github.com/MESH-Research/CCR/blob/master/.github/CONTRIBUTING.md) to help make CCR a reality.
+  CCR is currently in the initial development stage and isn't ready for deployment or testing just yet.  
```

Comment:
It's misleading to say 'CCR isn't ready for testing' when unit and integration tests have been implemented, but I respect if a different meaning was intended and there's a preference to change this to something else or preserve it.
```suggestion
  CCR is currently in the initial development stage and isn't ready for deployment just yet.  
```

Reply:
```suggestion
  CCR is currently in the initial development stage and isn't ready for deployment or evaluation just yet.  
```

Reply:
That works